walkthroughs/0-intro.md

```diff
@@ -1,27 +1,32 @@
-# Why Runme?
+---
```

Comment:
Remove frontmatter from all walkthrough files since VS Code will renderer them weirdly:

<img width="1769" alt="image" src="https://github.com/stateful/vscode-runme/assets/250527/b3c95fea-05dd-4642-9a63-50675560178e">